.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.11
+        environment:
+          GO111MODULE: "on"
+    steps:
+      - checkout
+      - run: go test -v -race -bench . ./...
```

Comment:
Better to use ginkgo instead of go test:
```
ginkgo -r -v --randomizeAllSpecs --randomizeSuites --trace --progress
```

Reply:
How do you run benchmarks with gingko?